cooltools/api/dotfinder.py

```diff
@@ -1663,7 +1682,8 @@ def dots(
     output_cols += [f"la_exp.{k}.qval" for k in kernels]
 
     # 4b. filter by enrichment and qval
-    if cluster_filtering is None:  # default - engage HiCCUPS filtering
+    if (cluster_filtering is None) or cluster_filtering:
```

Comment:
cluster_filtering=True makes sense ! let it be another option - it is confusing otherwise

so, now - `None` and `True` enable this filtering, while `False` deactivates it

Reply:
@gfudenberg could you look into this `cluster_filtering` flag - we've discussed it together - but last time I used it , it seemed very confusing to me, so I changed its behavior ...

You were suggesting `cluster_filtering=None` - to be default and engage filtering, and `cluster_filtering=False` to switch filtering off. By mistake tried `cluster_filtering=True` and it all crashed in a non-intuitive way

i just think it is worth keeping `cluster_filtering=True` functional